proto/file.pb.go

```diff
@@ -21,7 +22,7 @@ var _ = math.Inf
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
+const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
```

Comment:
Revert compilation of the protobuf. And in general get rid of all the `protobufjs` stuff you added in this commit, it isn't used anywhere and you just added more compiled files.

Reply:
done.